database/migrations/1607098837428_contact_data_schema.js

```diff
@@ -5,18 +5,46 @@ const Schema = use('Schema')
 
 class ContactDataSchema extends Schema {
   up () {
-    this.create('contact_data', (table) => {
+    this.createIfNotExists('users', (table) => {
```

Comment:
Hmmm... you moved all the migrations into one file? That's unusual, but if we're keeping that, then better to rename the file (from contact_data_schema to something like database_schema).

Reply:
Yeah. The problem was that, the contact schema was getting run first, and it relied on an id from the user table that didn't already exist